code/game/shuttles/vox.dm

```diff
@@ -63,6 +71,18 @@ var/global/datum/shuttle/vox/vox_shuttle = new(starting_area=/area/shuttle/vox/s
 	link_to(vox_shuttle)
 	.=..()
 
+	var/datum/holomap_marker/newMarker = new()
+	newMarker.id = "skipjack"
+	newMarker.icon = 'icons/holomap_markers_32x32.dmi'
+	newMarker.filter = HOLOMAP_FILTER_VOX
+	newMarker.x = x
+	newMarker.y = y
+	newMarker.z = z
+	newMarker.pixel_x = -16
```

Comment:
@clusterfack I said literally two posts above that it's not necessary because it's just a datum's var and the pixel multiplier is applied when the marker is added onto the map.